pkg/skaffold/runner/v1/dev.go

```diff
@@ -226,6 +228,22 @@ func (r *SkaffoldRunner) Dev(ctx context.Context, out io.Writer, artifacts []*la
 			continue
 		}
 
+		var absMatches []string
+		// Add absolute typed rules
+		if artifact.Sync != nil {
+			for _, pt := range artifact.Sync.Manual {
+				if pt.Type == "absolute" {
```

Comment:
Would it be possible to have this be `filepath.IsAbs(pt.Src)` and perhaps the `type` field is not needed?